userguide/kaui/includes/tenants.adoc

```diff
@@ -1,54 +1,338 @@
+* *Amount*—The cost of the plan in the new currency you're adding.
+* *Currency*—The currency you are adding.
++
+. Click the *Save* button. Kaui displays the new currency on the Catalog Show tab.
+
+==== Upload an XML Catalog
+
+_Something I'm not sure of is if you create a simple catalog, can you overwrite it by uploading an XML file? Seems like you should be able to do this, but I get an error when I attempt it. First, I get the Cloudflare error if I haven't removed the <?xml version="1.0" encoding="UTF-8" standalone="no"?> on the first line. After I clear that, I still get the KB error: "Error while communicating with the Kill Bill server: Invalid catalog for tenant: [TENANT #]." When I upload the same catalog on a new tenant, I don't have this problem. It loads fine._
+
+[NOTE]
+*Note:* To ensure a successful catalog file upload, check its validity with the Kill Bill catalog validation tool https://cloud.killbill.io/tools/catalog[here].
+
+To upload an XML catalog in Kaui:
+
+. <<View Catalog Products and Plans,Go to the Catalog Show tab>> on the Tenant Configuration page.
++
+image::Catalog-Tabs-ShowCat.png[width=850,align="center"]
++
+. Click the plus sign ( image:i_PlusGreen.png[] ) next to *Existing Plans*.
++
+Kaui opens the Catalog Configuration screen:
++
+image::CatalogConfiguration.png[width=450,align="center"]
++
+. Click on *Enable Advanced Configuration (Upload XML)*.
++
+Kaui displays an upload screen:
++
+image::Catalog-AdvancedConfigUpload.png[width=450,align="center"]
++
+. Click the *Choose File* button, locate the XML file, and select it.
++
+Once you have selected the file, Kaui displays the filename next to the *Choose File* button.
++
+image::Catalog-AdvancedConfigUpload-File.png[width=450,align="center"]
++
+. Click the *Upload* button.
++
+If the upload is successful, Kaui displays a confirmation message along with a list of plans on the Catalog Show tab:
++
+image::Catalog-UploadSuccess.png[width=950,align="center"]
+
+==== View the Catalog as XML
+
+. <<View Catalog Products and Plans,Go to the Catalog Show tab>> on the Tenant Configuration page.
++
+image::Catalog-Tabs-ShowCat.png[width=850,align="center"]
++
+. Click *Enable XML View*.
++
+Kaui displays the a list of catalog versions:
++
+image::Catalog-ListOfVersions.png[width=850,align="center"]
++
+. Click *view xml*. Kaui displays the raw XML (uneditable in this view).
++
+image::Catalog-RawXML.png[width=650,align="center"]
++
+. To return to Kaui, click the Back arrow button of your browser.
+
+==== Delete a Catalog
+
+_When I click delete catalog, I'm taken to the Kaui home page. So, I assume I don't have permission to delete the catalog?_
+
+. <<View Catalog Products and Plans,Go to the Catalog Show tab>> on the Tenant Configuration page.
++
+image::Catalog-Tabs-ShowCat.png[width=850,align="center"]
++
+. Click the red X ( image:i_RedX.png[] ) to the left of *Enable XML View*.
++
+_What is supposed to happen here?_
+
```

Comment:
The tenant configuration screen is shown with a **"Catalog was successfully deleted"** message. The catalog plans are also removed from the **Catalog Show** tab:
![delete_catalog](https://user-images.githubusercontent.com/85998496/162377140-65f4e8ea-e97b-478b-8d0d-db8db9c9a820.png)

Reply:
Fixed - see page 12.